packages/typescript-estree/src/useProgramFromProjectService.ts

```diff
@@ -1,4 +1,4 @@
-import type { ProjectServiceAndMetadata as ProjectServiceAndMetadata } from '@typescript-eslint/project-service';
+import type { ProjectServiceAndMetadata } from '@typescript-eslint/project-service';
```

Comment:
[Praise] Ha, yeah, one of these days we'll enable this lint rule...

Reply:
This is the only place where a useless rename is present in the code base.
I can enable the rule if you want. Here or in another PR